.github/workflows/ci-cd-pipeline.yml

```diff
@@ -137,6 +144,13 @@ jobs:
       TEST_ACCOUNT_ADDRESS: ${{ secrets.TEST_ACCOUNT_ADDRESS }}
       TEST_ACCOUNT_PRIVATE_KEY: ${{ secrets.TEST_ACCOUNT_PRIVATE_KEY }}
 
+  staging-starknet-go-tests:
+    needs: [promote_to_staging]
+    uses: ./.github/workflows/starknet-go-tests.yml
+    secrets:
+      TEST_RPC_URL: ${{ secrets.STAGING_SEPOLIA_URL }}/v0_7
+      TEST_ACCOUNT_PRIVATE_KEY: ${{ secrets.TEST_ACCOUNT_PRIVATE_KEY }}
```

Comment:
Are we planning to have different test accounts depending on the environment?  If no, I'd say to remove this as argument and just use in the starknet-go-tests the secret directly :)

Reply:
Yep, would be great to have different accounts per env in the future, as right now we can't run them parallel at all, because some of tests might fail(e.g due to invalid nonce while adding the txn)